nginx/assets/monitors/4xx.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "[NGINX] 4xx Errors",
```

Comment:
Can we change the title to something more descriptive like "4xx Errors higher than normal"?

Reply:
Updated